### PR TITLE
student2-edits

### PR DESCRIPTION
**Pull request by DanaiSk**
Hi student3-edits, can you review my changes?

### CODE REVIEW

File: index.html2.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Github Extra Practice</title>
+	<link href="./css/style.css" rel="stylesheet">
+</head>
```
Comment (DanaiSk) [inline index.html2.html:6]: I like what you did here

Comment (riannescholtens) [reply index.html2.html:6]: Thanks

File: index.html2.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<body>
+	<h1>WE L&#9829;VE GITHUB</h1>
+	<h2>We are Github greatest fans</h2>
+	<ul>
+		<li class="ct">Hi I'm <a href="#teacher">Laura</a></li>
+		<li class="cs1"><!-- Student 1 insert your name here and a link to the <div> container with id="student1" --></li>
```
Comment (Fleurvhoof) [inline index.html2.html:12]: Hola, it looks good!

**Review by Fleurvhoof — APPROVED** [review]

**Comment by riannescholtens** [comment]
Looks good

**Pull request merged**